CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### To be Released
 
+### 1.26.0
+
 * feat(db-commands) Add `database-enable-feature` and `database-disable-feature` for database addons [#807](https://github.com/Scalingo/cli/pull/807)
```

Comment:
Nothing about windows then?

Reply:
As it's an internal change we didn't added an entry. Do you want me to add one?

Reply:
Moreover it does not seem to actually fix the issue...

Reply:
It depends if it would fix any issue or not

Reply:
That's the odd thing. On @QuentinEscudierScalingo Windows computer, it seems it fixes the issue. On a Outscale Windows server, it does not fix the issue. So.... 🤷🏾

Reply:
Arf..